docs/conf.py

```diff
@@ -15,17 +15,18 @@
 # Incase the project was not installed
 import os
 import sys
+sys.path.append(os.path.abspath('../..'))
 sys.path.insert(0, os.path.abspath('..'))
 
 import mdakit_membcurv
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'MDAkit MembCurv'
-copyright = ("2021, MDAnalysis. Project structure based on the "
+project = 'MDAkit_membcurv'
```

Comment:
Just to keep consistent, could you please choose one name between `membrane-curvature` and `MDAkit_membcurv`? My preference is for the former but it's up to you.

Reply:
If it's ok, can I instead add an issue fully dedicated to rename `MDAkit MemCurv` to `membrane-curvature`? That change was included in the previous PR and got very messy.

Reply:
Of course :)

Reply:
yay, thanks! ☺️